examples/src/main/java/org/spine3/examples/aggregate/ClientApp.java

```diff
@@ -0,0 +1,246 @@
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.examples.aggregate;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.stub.StreamObserver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.spine3.base.Command;
+import org.spine3.base.Identifiers;
+import org.spine3.base.Queries;
+import org.spine3.base.Response;
+import org.spine3.client.CommandFactory;
+import org.spine3.client.Subscription;
+import org.spine3.client.SubscriptionUpdate;
+import org.spine3.client.Target;
+import org.spine3.client.Topic;
+import org.spine3.client.grpc.CommandServiceGrpc;
+import org.spine3.client.grpc.SubscriptionServiceGrpc;
+import org.spine3.examples.aggregate.command.AddOrderLine;
+import org.spine3.examples.aggregate.command.CreateOrder;
+import org.spine3.examples.aggregate.command.PayForOrder;
+import org.spine3.money.Money;
+import org.spine3.protobuf.AnyPacker;
+import org.spine3.protobuf.Messages;
+import org.spine3.time.ZoneOffsets;
+
+import java.util.List;
+
+import static com.google.common.collect.Lists.newLinkedList;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.spine3.client.ConnectionConstants.DEFAULT_CLIENT_SERVICE_PORT;
+import static org.spine3.client.UserUtil.newUserId;
+import static org.spine3.money.Currency.USD;
+import static org.spine3.money.MoneyUtil.newMoney;
+import static org.spine3.protobuf.Messages.toText;
+
+/**
+ * Sample of a client implementation.
+ *
+ * @author Mikhail Melnik
+ * @author Mikhail Mikhaylov
+ * @author Alexander Litus
+ */
+@SuppressWarnings("OverlyCoupledClass")     // OK for a self-contained all-in-one example.
+public class ClientApp {
+
+    @SuppressWarnings("DuplicateStringLiteralInspection")
+    private static final String SERVICE_HOST = "localhost";
+
+    private static final String RPC_FAILED = "RPC failed";
+    private static final int SHUTDOWN_TIMEOUT_SEC = 5;
+
+    private final CommandFactory commandFactory;
+    private final ManagedChannel channel;
+    private final CommandServiceGrpc.CommandServiceBlockingStub blockingClient;
+    private final SubscriptionServiceGrpc.SubscriptionServiceStub subscriptionClient;
+
+    private final StreamObserver<SubscriptionUpdate> orderUpdateObserver = new StreamObserver<SubscriptionUpdate>() {
+        @Override
+        public void onNext(SubscriptionUpdate subscriptionUpdate) {
+            final String updateText = Messages.toText(subscriptionUpdate);
+            log().info(" + Order updated: {}", updateText);
+        }
+
+        @Override
+        public void onError(Throwable throwable) {
+            log().error("Subscription update delivery error occurred", throwable);
+        }
+
+        @Override
+        public void onCompleted() {
+            log().info("Subscription update delivery completed.");
+        }
+    };
+
```

Comment:
To be processed along with `examples` project update after `core-java` 0.6.0 release.